src/pages/home/ReportScreen.js

```diff
@@ -76,25 +88,39 @@ class MainView extends Component {
                 || report.reportID === activeReportID
         ));
         return (
-            <>
-                {_.map(reportsToDisplay, report => (
-                    <View
-                        key={report.reportID}
-                        style={reportStyles[report.reportID]}
-                    >
-                        <ReportView
-                            reportID={report.reportID}
-                            isActiveReport={report.reportID === activeReportID}
+            <ScreenWrapper
+                style={[
+                    styles.appContent,
+                    styles.flex1,
+                    styles.flexColumn,
+                ]}
+            >
+                {() => (
```

Comment:
I can see how it might not be super familiar to everyone. One caveat here is that if you even try to use it in any other way then it will throw a `propTypes` warning on dev and also crash everything. So in a sense we are documenting that already and protecting against it here:

https://github.com/Expensify/Expensify.cash/blob/1b444ecd0005dd923d3626c66d4faf9826244f2c/src/components/ScreenWrapper.js#L11-L12

Or did you have something else in mind?

Reply:
Another idea I had was to allow both a function as a child and not, but I wasn't sure if it would be necessary yet. In this case, we are not using the insets so maybe that could be a future improvement.

Reply:
Ahh, it wasn't intuitive while I was reviewing at first, but it did crash with that prop-types error when I tried to change it locally. I think when people are developing they will realize it the same way I did so I think that's fine.

> Another idea I had was to allow both a function as a child and not
I'm not familiar with how this might look, but it _sounds_ like it might be more confusing that simply using `() => `, so I think it's a good call to save that for a future improvement